src/core/request/deltafetch.spec.js

```diff
@@ -1,316 +0,0 @@
-import nock from 'nock';
```

Comment:
So now we only have one test for delta set? Will there be more added?

Reply:
I added 2 more for `pull()`. I will probably add some for `deltaSet.js`